cfgov/jinja2/know-before-you-owe/index.html

```diff
@@ -65,7 +65,7 @@ <h1 class="m-hero_heading">Know Before You Owe | Mortgages</h1>
 {% endblock %}
 
 {% block body_content %}
-<div class="wrapper wrapper__match-content">
```

Comment:
There are a couple [kbyo](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/jinja2/know-before-you-owe/timeline/index.html) [sub-templates](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/jinja2/know-before-you-owe/compare/index.html) that probably also need updating?